scripts/minikube.sh

```diff
@@ -176,6 +177,10 @@ else
     DISK_CONFIG=""
 fi
 
+if [[ -n "${MINIKUBE_ISO_URL}" ]]; then
+    EXTRA_CONFIG="${EXTRA_CONFIG} --iso-url ${MINIKUBE_ISO_URL}"
+fi
```

Comment:
Need to check this works with podman driver as we are moving to podman driver to create kubernetes cluster

Reply:
we can consider running a daily job that runs on a Amazon bare-metal system and deploy minikube with its VM there. But for regular CI jobs, we'll rely on Amazon VMs for now.

Reply:
I just started a CentOS 8 Stream VM, it has kernel 4.18. That is a bit too old to properly run fscrypt. Ideally this would be >= 5.4 [0].

Something bare metal would be awesome! The Ceph FS fscrypt support would  require something even newer..

[0] https://github.com/google/fscrypt#runtime-dependencies

Reply:
How to move forward? Feature flag the file encryption tests?